metrics/server.go

```diff
@@ -112,6 +113,13 @@ func NewServerMetrics(c ServerConfig) *ServerMetrics {
 			Help:      "The current status of the server (0 => New, 1 => Running, 2=> Closing, 3 => closed)",
 		}),
 
+		BroadcastClientCount: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: "goomerang",
+			Subsystem: "server",
+			Name:      "broadcasts_client_total",
```

Comment:
make the singular form "broadcasts_client_total" instead of "broadcast_client_total". Be consistent.

Reply:
Fixed at c15798d259e09890b0158a20b1823359813f2beb